docs/tutorials/stateless-application/hello-minikube.md

```diff
@@ -291,6 +291,52 @@ Run your app again to view the new message:
 minikube service hello-node
 ```
 
+## Enable Add-ons
+
+Minikube has a set of built in addons that can be used enabled, disabled, and opened inside of the local Kubernetes environment. Below is an example of this functionality for the `heapster` addon.
```

Comment:
be used enabled, disable ?

Reply:
"Below is an example of this functionality for the `heapster` addon." this can be removed.

Reply:
I would suggest this:
Minikube has a set of built-in addons that can be enabled or disabled in the local Kubernetes environment.

Reply:
@tengqm This is Minikube subcommands for addons.